actionmailer/test/delivery_methods_test.rb

```diff
@@ -102,16 +102,21 @@ def welcome(hash={})
   end
 
   test "ActionMailer should be told when Mail gets delivered" do
-    DeliveryMailer.expects(:deliver_mail).once
-    DeliveryMailer.welcome.deliver_now
+    DeliveryMailer.delivery_method = :test
```

Comment:
Shouldn't delivery method be test by default? I'd rather if we found a way not to set it.